test/cppsuite/tests/test_live_restore.cpp

```diff
@@ -104,12 +106,19 @@ class database_model {
     std::vector<std::string> _collections;
 };
 
-static const int crud_ops = 20000;
-static const int warmup_insertions = crud_ops / 3;
+static const int iteration_count_default = 2;
+/*
+ * FIXME-WT-13825: Set thread_count_default to non zero once extent list concurrency is implemented.
+ */
+static const int thread_count_default = 0;
+static const int op_count_default = 20000;
+static const int warmup_insertion_factor = 3;
```

Comment:
What's the purpose of this?

Reply:
The insertion factor? Effectively the first time we create the database we intentionally do a number of insertions to get things started. The factor defines how many insertions we want to do prior to starting the regular CRUD operations. It also only applies to the first iteration.

Reply:
Okay I see, thanks!